src/cpp/session/modules/SessionHelp.R

```diff
@@ -797,5 +793,28 @@ options(help_type = "html")
 
 .rs.addJsonRpcHandler("follow_help_topic", function(url)
 {
-   .rs.followHelpTopic(url)
+   rx <- "^.*/help/library/([^/]*)/help/(.*)$"
+   pkg <- sub(rx, "\\1", url)
+   topic <- utils::URLdecode(sub(rx, "\\2", url))
+
+   .rs.followHelpTopic(pkg = pkg, topic = topic)
+})
+
+.rs.addFunction("Rd2HTML", function(file, package = "", Rdmacros = "")
+{
+   tf <- tempfile(); on.exit(unlink(tf))
+   macros <- suppressWarnings(tools:::initialRdMacros(Rdmacros))
+   tools::Rd2HTML(file, out = tf, package = package, macros = macros, dynamic = TRUE)
+   lines <- readLines(tf, warn = FALSE)
+   lines <- sub("R Documentation</td></tr></table>", "(preview) R Documentation</td></tr></table>", lines)
+   if (nzchar(package))
+   {
+      # does not work, need to find a way to serve the figures from the dev directory
+      # lines <- sub('img src="figures/', sprintf('img src="help/library/%s/html/figures/', package), lines, fixed = TRUE)
```

Comment:
Do you want to address this in this PR or separately?

I think you could either try other tweaks to the HTML, or consider adding a custom handler for these things here:

https://github.com/rstudio/rstudio/blob/ddf4818473358a5eddeb6af375439db2c36488b7/src/cpp/session/modules/SessionHelp.cpp#L808-L865

Reply:
This is where it happens in `httpd`: 

https://github.com/r-devel/r-svn/blob/b9c241d06c43993184c8e686575dac44b382ff47/src/library/tools/R/dynamicHelp.R#L650-L654

I'll try to deal with this here.